music21/tablature.py

```diff
@@ -344,6 +352,17 @@ def __init__(self, fretNotes=None, displayFrets=4):
         super().__init__(numStrings, fretNotes, displayFrets)
 
         self.tuning = [pitch.Pitch('G3'), pitch.Pitch('D4'), pitch.Pitch('A4'), pitch.Pitch('E5')]
+
+class CustomFretBoard(FretBoard):
+    '''
+    A custom fretboard tuned with any list of pitches
+    '''
+
+    def __init__(self, pitches, fretNotes=None, displayFrets=4):
+        numStrings = len(pitches)
+        super().__init__(numStrings, fretNotes, displayFrets)
+
+        self.tuning = pitches.copy()
```

Comment:
I'm not certain we need a new class. The only thing I see different compared to `FretBoard` is being able to pass pitches to the constructor instead of doing:

```python
fb = FretBoard(numStrings=len(pitches))
fb.pitches = pitches  # or pitches.copy()
```

And at this point I would be reluctant to create a new public class for that, when we just need to solve musicxml translation. But let me know if there are some details I've neglected here.

Reply:
agreed.